inference-engine/CMakeLists.txt

```diff
@@ -89,11 +89,6 @@ endfunction()
 # they must be built even if samples build is disabled (required for tests and tools).
 ie_build_samples()
 
-file(GLOB_RECURSE SAMPLES_SOURCES samples/*.cpp samples/*.hpp samples/*.h)
-add_cpplint_target(sample_cpplint
```

Comment:
Am I right that it is a legacy code? And for samples we still have code style check?

Reply:
yes, code style for sample is in samples/CMakeLists.txt